bpf/lib/bpf_helpers.h

```diff
@@ -65,6 +65,21 @@ enum bpf_enum_value_kind {
 	BPF_ENUMVAL_VALUE = 1, /* enum value value relocation */
 };
 
+/*
```

Comment:
why is this needed in this commit?

Reply:
Because `bpf_core_read` calls [`bpf_probe_read`](https://github.com/cilium/tetragon/blob/bf80f7097f19efa617a53b919a3cf22cbbed3bc9/bpf/libbpf/bpf_core_read.h#L117), which is not defined in `bpf_helpers.h`. In tetragon, we once used a customized macro [`_(P)`](https://github.com/cilium/tetragon/blob/bf80f7097f19efa617a53b919a3cf22cbbed3bc9/bpf/lib/bpf_helpers.h#L51) to replace it. But it is not the "libbpf way".